include/session/types.hpp

```diff
@@ -1,13 +1,23 @@
 #pragma once
 
+#include <array>
 #include <cstdint>
-#include <span>
-#include <string>
-#include <string_view>
-#include <vector>
 
-namespace session { namespace config {
+namespace session {
+using array_uc32 = std::array<std::uint8_t, 32>;
+using array_uc33 = std::array<std::uint8_t, 33>;
+using array_uc64 = std::array<std::uint8_t, 64>;
 
-    using seqno_t = std::int64_t;
+enum class SessionIDPrefix {
+    standard = 0,
+    group = 0x3,
+    community_blinded_legacy = 0x5,
+    community_blinded = 0x15,
+    version_blinded = 0x25,
+    unblinded = 0x7,
+};
```

Comment:
I am very confused by this. Is it supposed to map to real prefixes?
If yes this should all be changed a fair bit:
```hpp
```suggestion
enum class SessionIDPrefix {
    standard = 0x05,
    group = 0x03,
    community_unblinded = 0x05,
    community_blinded_legacy = 0x15,
    community_blinded = 0x25,
    version_blinded = 0x07,
};
```

I guess 0x3 is actually 0x03, but the prefixes/names associated are not correct too